internal/server/v2/facet/contained_in.go

```diff
@@ -143,19 +146,42 @@ func ContainedInFacet(
 					for _, entity := range childPlaces {
 						if facetData, ok := entityData.ByEntity[entity]; ok {
 							for _, item := range facetData.OrderedFacets {
-								if _, ok := seenFacetId[item.FacetId]; ok {
-									continue
+								// obsCount is the number of entities with data for this facet
+								obsCount := int32(1)
```

Comment:
obsCount := 1 should work

Reply:
it does not because FacetObservation requires ObsCount to be int32